plugins/nirmata/plugin.go

```diff
@@ -0,0 +1,22 @@
+package nirmata
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "nirmata",
+		Platform: schema.PlatformInfo{
+			Name:     "Nirmata",
+			Homepage: sdk.URL("https://nirmata.com"), // TODO: Check if this is correct
```

Comment:
It looks like nirmata.com and nirmata.io are two different products/services, but from the same company? Could you share some specifics of what you know about Nirmata? That'd help us reviewers jumpstart the process, vs having to understand the Nirmata concepts manually. Thanks @itsCheithanya!